### PR TITLE
Add methods for sortable attributes settings

### PR DESCRIPTION
**Pull request by albajarazeth**
# Pull Request

## Related issue
Fixes Issue #491 

## What does this PR do?

-  getSortableAttributesSettings
-  updateSortableAttributesSettings
-  resetSortableAttributesSettings
-  Add tests
-  Add sample in the code-sample file

## PR checklist
Please check if your PR fulfills the following requirements:
- [yes] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [yes] Have you read the contributing guidelines?
- [yes] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: src/test/java/com/meilisearch/integration/SettingsTest.java [hunk 1]
```diff
@@ -522,6 +522,64 @@ public void testResetFilterableAttributesSettings() throws Exception {
         assertNotEquals(updatedFilterableAttributes.length, filterableAttributesAfterReset.length);
         assertNotEquals(initialFilterableAttributes.length, updatedFilterableAttributes.length);
     }
+    /** Tests of the sortable attributes setting methods**/
+
+    @Test
+    @DisplayName("Test get sortable attributes settings by uid")
+    public void testGetSortableAttributesSettings() throws Exception {
+        Index index = createIndex("testGetDisplayedAttributesSettings");
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/SettingsTest.java:530]:
```suggestion
        Index index = createIndex("testGetSortableAttributesSettings");
```

File: src/test/java/com/meilisearch/integration/SettingsTest.java [hunk 2]
```diff
@@ -522,6 +522,64 @@ public void testResetFilterableAttributesSettings() throws Exception {
...
+
+        assertEquals(
+            initialSettings.getSortableAttributes().length,
+            initialSortableAttributes.length);
+        assertArrayEquals(initialSettings.getSortableAttributes(), initialSortableAttributes);
+    }
+    @Test
+    @DisplayName("Test update sortable attributes settings")
+    public void testUpdateSortableAttributesSettings() throws Exception {
+        Index index = createIndex("testUpdateDisplayedAttributesSettings");
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/SettingsTest.java:543]:
```suggestion
        Index index = createIndex("testUpdateSortableAttributesSettings");
```

File: src/test/java/com/meilisearch/integration/SettingsTest.java [hunk 3]
```diff
@@ -522,6 +522,64 @@ public void testResetFilterableAttributesSettings() throws Exception {
...
+        assertThat(
+            Arrays.asList(newSortableAttributes),
+            containsInAnyOrder(updatedSortableAttributes));
+        assertNotEquals(initialSortableAttributes.length, updatedSortableAttributes.length);
+    }
+
+    @Test
+    @DisplayName("Test reset sortable attributes settings")
+    public void testResetSortableAttributesSettings() throws Exception {
+        Index index = createIndex("testUpdateDisplayedAttributesSettings");
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/SettingsTest.java:561]:
```suggestion
        Index index = createIndex("testUpdateSortableAttributesSettings");
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -641,10 +639,9 @@ sorting_guide_sort_parameter_2: |-
 get_sortable_attributes_1: |-
   client.index("books").getSettings().getSortableAttributes();
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:640]:
```suggestion
get_sortable_attributes_1: |-
  client.index("books").getSortableAttributesSettings();
```

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -662,7 +659,7 @@ geosearch_guide_filter_usage_2: |-
   client.index("restaurants").search(searchRequest);
 geosearch_guide_sort_settings_1: |-
   Settings settings = new Settings();
-  settings.setSortableAttributes(new String[] {"_geo"});
+  client.index("restaurants").updateSortableAttributesSettings(new String[] {"_geo"});
   client.index("restaurants").updateSettings(settings);
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:663]: Should be replaced by just this line:
```yaml
geosearch_guide_sort_settings_1: |-
  client.index("restaurants").updateSortableAttributesSettings(new String[] {"_geo"});
```

**Review by alallema — COMMENTED** [review]
Thanks a lot for this PR @albajarazeth ❤️ 
Great job, just there is some linting issues you need to run the linter with this command:
```sh
➜ ./gradlew spotlessApply
```
I also think that you are not updated with the main branch because this code-sample should be modified too:
```java
date_guide_sortable_attributes_1: |-
  client.index("games").updateSortableAttributesSettings(new String[] {"release_timestamp", "author"});
```
> I'm not completely certain I put the code samples correctly on the yaml file

And don't worry about it; it's already great 🎉

File: .code-samples.meilisearch.yaml (hunk 5 above)
Comment (albajarazeth) [reply .code-samples.meilisearch.yaml:663]: Thank you! I will fix it.

**Pull request closed without merging**